window.guardian.d.ts

```diff
@@ -34,7 +34,14 @@ declare global {
         };
         GoogleAnalyticsObject: string;
         ga: UniversalAnalytics.ga;
-        initAutomatJs?: Function;
+
+        // TODO expose as type from Automat client lib
+        automat: {
+            react: any;
```

Comment:
Is there any way to cast these to something more specific, ideally including definitions from the actual libraries? I can't think of a quick way of doing that but there might be one. It would stop anybody assigning the wrong library to any of these vars (it could even go further and add some guard against versioning).

Reply:
Yeah agree this would be nice. I think these could be good things to add to the Automat client lib if/when we have fully validated this approach.